run_conformance_tests.py

```diff
@@ -90,15 +95,24 @@ def handle_args(argv):
 
     host_names = (args.host_platform_name, args.host_device_name)
     opencl_names = (args.opencl_platform_name, args.opencl_device_name)
+    full_conformance = 'OFF' if args.fast else 'ON'
 
     return (args.cmake_exe, args.build_system_name, args.build_system_call,
-            args.conformance_filter, args.implementation_name,
+            full_conformance, args.conformance_filter, args.implementation_name,
             args.additional_cmake_args, host_names, opencl_names,
             args.additional_ctest_args, args.build_only)
 
-
-def generate_cmake_call(cmake_exe, build_system_name, conformance_filter,
-                        additional_cmake_args, host_names, opencl_names):
+def split_additional_args(additional_args):
+    if additional_args is None:
+        return []
+    if os.path.sep is '\\':
+        return shlex.split(additional_args.replace('\\','\\\\'))
```

Comment:
I have the feeling that even on Unix if you have a `\` this is a special character and you have to escape it with... `\`!
It is not clear to me why not using for example `shlex.quote()` here.

Reply:
Yes, ``shlex.quote()`` fits much better. I've missed it whilst trying to fix passing any windows path within ``additional-cmake-args``. Thanks.